routers/posts.py

```
@@ -1,42 +1,42 @@
+from typing import Annotated, List, Optional
```

Ruff (0.14.13)

`typing.List` is deprecated, use `list` instead

(UP035)